test_mavros/sitl_test/launch/px4/iris_empty_world_offboard_posctl_square.launch

```diff
@@ -0,0 +1,59 @@
+<launch>
+    <!-- vim: set ft=xml noet : -->
+    <!-- Offboard Position Control - square path - launch file -->
+    
+    <arg name="headless" default="false"/>
+    <arg name="gui" default="true"/>
+    <arg name="enable_logging" default="false"/>
+    <arg name="enable_ground_truth" default="true"/>
+    <arg name="ns" default="iris"/>
+    <arg name="log_file" default="$(arg ns)"/>
+
+    <!-- Gazebo simulation and vehicle model launch -->
+    <include file="$(find rotors_gazebo)/launch/iris_empty_world.launch">
+        <arg name="headless" value="$(arg headless)"/>
+        <arg name="gui" value="$(arg gui)"/>
+        <arg name="enable_logging" value="$(arg enable_logging)" />
+        <arg name="enable_ground_truth" value="$(arg enable_ground_truth)" />
+        <arg name="log_file" value="$(arg log_file)"/>
+    </include>
+    
+    <include file="$(find px4)/launch/iris.launch">
+        <arg name="ns" value="$(arg ns)"/>
+    </include>
```

Comment:
One bad thing that we can't change is absence rotors_simulator for jade. And PX4 too, so we can't add it to package.xml.
Only may try as `<run_depend>`.